tests/php/i18n/i18nTextCollectorTest.php

```diff
@@ -712,11 +712,11 @@ public function testModuleDetection()
         $collector = new Collector();
         $modules = ModuleLoader::inst()->getManifest()->getModules();
         $this->assertEquals(
-            array(
+            [
                 'i18nnonstandardmodule',
+                'i18nothermodule',
```

Comment:
Why has the order changed here? I assume that doesn't change the YAML config order, since that's determined independently of module loading order. Is there anything else that relies on it though?

Reply:
Alphabetisation is the default; I'm not sure WHY it was the way it was earlier. Perhaps it wasn't alphabetised properly?

Note: Module sorting is still applied after this and can be controlled via config.